cw-orch/Cargo.toml

```diff
@@ -12,42 +12,26 @@ readme = "../README.md"
 
 exclude = [".env"]
 
+[[example]]
+name = "ica-demo"
+required-features = ["interchain"]
+
+# Interact with local/remote daemons over gRPC
 [features]
 # enable node-backed tests (ensure Docker is running)
 # run with `cargo test --jobs 1 --features node-tests`
 node-tests = ["daemon"]
 # enable the optional dependencies
-daemon = [
-  "dep:ibc-chain-registry",
-  "dep:ibc-relayer-types",
-  "dep:tokio",
-  "dep:tonic",
-  "dep:secp256k1",
-  "dep:reqwest",
-  "dep:base64",
-  "dep:hkd32",
-  "dep:rand_core",
-  "dep:ed25519-dalek",
-  "dep:cosmrs",
-  "dep:derive_builder",
-  "dep:eyre",
-  "dep:chrono",
-  "dep:base16",
-  "dep:ripemd",
-  "dep:ring",
-  "dep:hex",
-  "dep:bitcoin",
-  "dep:prost",
-  "cw-orch-contract-derive/propagate_daemon",
-]
+daemon = ["dep:ibc-chain-registry", "dep:ibc-relayer-types", "dep:tokio", "dep:tonic", "dep:secp256k1", "dep:reqwest", "dep:base64", "dep:hkd32", "dep:rand_core", "dep:ed25519-dalek", "dep:cosmrs", "dep:derive_builder", "dep:eyre", "dep:chrono", "dep:base16", "dep:ripemd", "dep:ring", "dep:hex", "dep:bitcoin", "dep:prost", "dep:sha256", "dep:log4rs", "dep:diff-struct", "dep:futures-util", "cw-orch-contract-derive/propagate_daemon"]
```

Comment:
are you using taplo for toml formatting? 

`find . -type f -iname "*.toml" -print0 | xargs -0 taplo format` formats all the tomls

Reply:
i have an alias for it, you don't need to run it all the time but CI checks for it (which is why your fmt job is failing). 

I think i added it in the just file (maybe was in contracts)

Reply:
added to just file in last version